File: 1894-merge-strings-alternately/1894-merge-strings-alternately.ts
function mergeAlternately(word1: string, word2: string): string {
    let result: string = ''
    let longest: number = word1.length > word2.length ? word1.length: word2.length;
    for(let i = 0; i < longest; i++){
        word1[i] != undefined ? result += word1[i]: null;
        word2[i] != undefined ? result += word2[i]: null;
    }
    return result
};

File: 1894-merge-strings-alternately/1894-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        res = []
        
        for i in range(len(word1)):
            if i < len(word1):
                res.append(word1[i])
            if i < len(word2):
                res.append(word2[i])
        res.append(word2[len(word1):])
        return ''.join(res)
        